docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+  The default is enabled for Release builds.
+
+- **Trimming granularity**
+
+  Controls how aggressively IL is discarded. There are two modes to select from:
+
+  - `Link` enables member-level trimming, which removes unused members from types.
+  - `CopyUsed` (default) enableds assembly-level trimming, which keeps an entire assembly if any part of it is used.
+
+- **Java max heap size**
+
+  Set this value to increase the size of memory that an app can use. For example, a value of `2G` increases the heap size to 2 gigabytes. Note that there isn't a guarantee of how large the heap will be, and requesting too much heap memory may force other apps to terminate prematurely.
+
+  The default is `1G`/
+
+- **Additional Java options**
+
+  Specifies additional command-line options to pass to the Java compiler when building a _.dex_ file. From the command line, you can type `java -help` to see the available options.
+
+### Package Signing
+
+When enabled, signs the _.APK_ file using the keystore details. This is disabled by default.
+
+## iOS
+
+These are iOS-specific .NET MAUI settings.
+
+### Build
+
+Settings related to building the iOS app.
+
+- **Linker behavior**
+
+  The linker can strip out unused methods, properties, fields, events, structs, and even classes in order to reduce the overall size of the application. You can add a `Preserve` attribute to any of these in order to prevent the linker from stripping it out if it's needed for serialization or reflection.
+
+  > [!WARNING]
+  > Enabling this feature may hinder debugging, as it may strip out property accessors that would allow you to inspect the state of your objects.
+
+  Options are:
+
+  - `Don't link`
+  - `Link Framework SDKs only` (default)
+  - `Link All`
+
+- **LLVM**
+
+  When enabled, uses the LLVM optimized compiler. This can be set for individually for both Debug and Release modes.
+
+  The default is enabled for Release builds.
+
+- **Float operations**
+
+  Performs all 32-bit float operations as 64-bit float operations.
+
+- **Symbols**
+
+  When enabled, strips native debugging symbols from the output. This is enabled by default.
+
+- **Garbage collector**
+
+  When enabled, uses the concurrent garbage collector. This is disabled by default.
+
+- **Additional arguments**
+
+  Additional command line arguments to be passed to the application bundling code.
+
+- **Optimization**
+
+  When enabled, optimizes _.PNG_ images. This is enabled by default.
+
+### Bundle Signing
+
+These settings are related to generating and signing the app bundle.
+
+- **Scheme**
+
+  Configures the signing scheme for the bundle. It can be set to one of the following values:
+
+  - `Manual provisioning`: With this value, you'll be responsible for setting provisioning profiles and signing certificates yourself.
+  - `Automatic provisioning`: (default) With this value, Visual Studio will set provisioning profiles and signing certificates for you to simplify app testing on a device.
```

Comment:
I think it would be better to say that automatic provisioning simplifies app deployment to a device (for testing).

Reply:
```suggestion
  - `Automatic provisioning`: (default) With this value, Visual Studio will set provisioning profiles and signing certificates for you, which simplifies app deployment when testing on a device.
```